### PR TITLE
New package: Microsoft.SqlPackage version 162.3.566

### PR DESCRIPTION
**Pull request by tristanbarcelon**
###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/167092)

### CODE REVIEW

File: manifests/m/Microsoft/SqlPackage/162.3.566/Microsoft.SqlPackage.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+PackageLocale: en-US
+Publisher: Microsoft Corporation
+PublisherUrl: https://learn.microsoft.com/en-us/sql/tools/sqlpackage/sqlpackage?view=sql-server-ver16
+Author: Microsoft Corporation
+PackageName: Microsoft SqlPackage
+PackageUrl: https://learn.microsoft.com/en-us/sql/tools/sqlpackage/sqlpackage?view=sql-server-ver16
+License: Proprietary
+Copyright: Copyright (c) Microsoft Corporation
+ShortDescription: SqlPackage is a command-line utility that automates the database development tasks by exposing some of the public Data-Tier Application Framework (DacFx) APIs.
+Description: qlPackage is a command-line utility that automates the database development tasks by exposing some of the public Data-Tier Application Framework (DacFx) APIs. The primary use cases for SqlPackage focus on database portability and deployments for the SQL Server, Azure SQL, and Azure Synapse Analytics family of databases. SqlPackage can be automated using Azure Pipelines and GitHub actions or other CI/CD tools.
```
Comment (stephengillie) [inline manifests/m/Microsoft/SqlPackage/162.3.566/Microsoft.SqlPackage.locale.en-US.yaml:15]:
```suggestion
Description: SqlPackage is a command-line utility that automates the database development tasks by exposing some of the public Data-Tier Application Framework (DacFx) APIs. The primary use cases for SqlPackage focus on database portability and deployments for the SQL Server, Azure SQL, and Azure Synapse Analytics family of databases. SqlPackage can be automated using Azure Pipelines and GitHub actions or other CI/CD tools.
```

Common copy-paste error.

**Comment by tristanbarcelon** [comment]
lgtm

**Comment by tristanbarcelon** [comment]
hi @stephengillie, is there anything else I need to do to have the package manifest merged?

**Comment by stephengillie** [comment]
Sorry for the delay. This package installs and launches normally. But the package manager creates a symlink instead of a `.LNK` file, leaving the symlink unable to access DLLs or other files that it needs.

image

**Comment by Trenly** [comment]
@tristanbarcelon - Support for zipped binaries has now been added and is available in the pipelines. This will require you to update your submission to ManifestVersion: 1.9.0 and set ArchiveBinariesDependOnPath: true in the installer node. To keep things clean, this PR is being closed. Please submit a new PR for the latest version of this package. Thank you for your contributions!

**Pull request closed without merging**